### PR TITLE
Add a basic README

### LINKED ISSUE #1: Fix README
The README is [currently](https://github.com/everypolitician/field_serializer/blob/202d7b26fefaa9699556481a772c2732d8721376/README.md) boilerplate.

### PR DESCRIPTION
**Pull request by chrismytton**
Fill out a basic README.

Fixes #1 
# Notes to reviewer

I struggled a bit to come up with a decent description of this gem. I'm not sure if that's because it's not solving a well enough defined problem or if it's just a bit late on a Friday but any suggestions for improvements are welcome!

### CODE REVIEW

**Comment by chrismytton** [comment]
@tmtmtmtm 👀

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # FieldSerializer
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/field_serializer`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+Ruby module which allows you to specify a set of fields that you'd like
+serialized in the `#to_h` method.
```
Comment (tmtmtmtm) [inline README.md]: My main comment here is that this is only really explaining the _what_, rather than the _why_.

Comment (chrismytton) [reply README.md]: Yep, good point! I've added a few more words on why we built it and how we're using it.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # FieldSerializer
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/field_serializer`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+Ruby module which allows you to specify a set of fields that you'd like
+serialized in the `#to_h` method. We ([EveryPolitician](http://everypolitician.org/))
+are using this library in our scrapers to make them more declaritive. It allows
```
Comment (tmtmtmtm) [inline README.md]: typo: declaritive → declarative

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
Tiny typo, and I suspect we're going to want to tweak this considerably over time, but it's definitely an improvement over the existing README, so otherwise 👍

File: README.md (hunk 2 above)
Comment (chrismytton) [reply README.md]: Oops! Fixed in https://github.com/everypolitician/field_serializer/pull/11/commits/be28beb9eff91aabea5872422f837cb62f0e6c95.

**Review by tmtmtmtm — APPROVED** [review]

**Pull request merged**